server/config/configresource/config_resource_test.go

```diff
@@ -0,0 +1,63 @@
+package configresource_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/gorilla/mux"
+	"github.com/kubeshop/tracetest/server/config/configresource"
+	"github.com/kubeshop/tracetest/server/id"
+	"github.com/kubeshop/tracetest/server/resourcemanager"
+	rmtests "github.com/kubeshop/tracetest/server/resourcemanager/testutil"
+	"github.com/kubeshop/tracetest/server/testmock"
+)
+
+func TestConfigResource(t *testing.T) {
+
+	db := testmock.MustGetRawTestingDatabase()
+	sampleConfig := configresource.Config{
+		ID:               "1",
+		Name:             "test",
+		AnalyticsEnabled: true,
+	}
+
+	rmtests.TestResourceType(t, rmtests.ResourceTypeTest{
+		ResourceType: "Config",
+		RegisterManagerFn: func(router *mux.Router) any {
```

Comment:
This function is run before each operation, so serves as a kind of `setup` function